src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/RecalcEngineTests.cs

```diff
@@ -711,6 +711,34 @@ public void ImperativeUserDefinedFunctionTest(string udfExpression, string expre
             }
         }
 
+        [Theory]
+
+        // Behavior function in imperative udf
+        [InlineData(

```

Comment:
>InlineData [](http://example.com/codeflow?start=9&length=10)

If there is only one case, then we can make it a [Fact] and inline the arguments.

We can also add some additional cases for things such as examples where the types are mismatched, but there is an implicit coercion that can make it work (e.g., `MyFunc(): Text = { 1; 2 };`)